### PR TITLE
fix: fixed the position of the dropdown menu and white-space of tbody

### PR DESCRIPTION
**Pull request by SaraBianchi**
folder contents bugfix
- fixed the position of the dropdown menu of each table row
- added min-height for the table to better show the dropdown menu even when there is only one item
- added style to wrap titles on desktop monitor

### CODE REVIEW

**Review by giuliaghisini — APPROVED** [review]

**Comment by giuliaghisini** [comment]
@tiberiuichim @sneridagh

File: theme/themes/pastanaga/extras/contents.less [hunk 1]
```diff
@@ -78,8 +78,14 @@
...
   }
 
+  .ui.upward.dropdown > .menu {
+    top: 100%;
+    bottom: inherit;
+  }
+
   .contents-table-wrapper {
     width: 100%;
+    min-height: 13.8888rem;
```
Comment (ichim-david) [inline theme/themes/pastanaga/extras/contents.less:88]: @SaraBianchi why not simply use 14rem?
Looking here https://volto.kitconcept.com/contents
**13.8888rem** amounts  to a subpixel value **194.443px**
while **14rem** equals to **196px**

Comment (SaraBianchi) [reply theme/themes/pastanaga/extras/contents.less:88]: I don't remember, maybe I made my calculations wrong. No problem, we can change it

**Comment by sneridagh** [comment]
@SaraBianchi Changelog is required! It seems the bot does not complain now.

**Review by sneridagh — APPROVED** [review]

**Comment by sneridagh** [comment]
@SaraBianchi I just realised of a regression because of this PR:

This should be the outcome of the folder contents:

image


And instead, now we get this:

image

Could you please take a look? Thanks!

**Comment by sneridagh** [comment]
Just added https://github.com/plone/volto/issues/3273 to keep track.

**Pull request merged**